packages/shared/src/components/modals/SquadMemberModal.tsx

```diff
@@ -122,8 +123,13 @@ export function SquadMemberModal({
           onScroll: hideMenu,
         }}
         userListProps={{
-          additionalContent: (user, index) => (
-            <SquadMemberItemAdditionalContent
+          additionalContent: (user, index) => [
```

Comment:
Not 100% sure if I like this array object flow, so let me know please 🤣 
It was just the easiest.

Reply:
Yeah not sure I like it either... 😅 

Had to dig in to see how we have composed this components. I think maybe ideally the UserShortInfo component should allow for all of these additional details to user if provided props to show it. Still there seems to be a lot of info being passed around until u end up to UserShortInfo so might need a better look at it all if we want to refactor. 

Saying that, if we want to move fast with it, think we're good. Let's just note it for further improvements.

Reply:
100% could probably do with a refactor.
In general it's not needing anything, only this squad one adds "for now" but expect it to have exceptions later on again.